projects/plugins/boost/compatibility/page-optimize.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Compatibility file for Page Optimize.
+ *
+ * This will synchronize the settings from Page Optimize to Jetpack Boost.
+ * It will also disable the Page Optimize functionality.
+ *
+ * @package automattic/jetpack-boost
+ */
+
+$js_concatenate_enabled  = (bool) get_option( 'page_optimize-js', page_optimize_js_default() );
+$boost_js_concat_enabled = get_option( 'jetpack_boost_status_minify-js' );
```

Comment:
`boost_js_concat_enabled` should probably be `boost_js_concat_exists` because you're checking whether the value exists in the database. And since types in the database in WordPress are a bit funky, I think this might need a comment.

Reply:
Updated in b35035327bd8384b4d1af5c36c61913c6727e952.